src/locales/translations.json

```diff
@@ -301,6 +336,46 @@
       "pt-br": "Filas",
       "en": "Queues",
       "es": "Colas"
+    },
+    "create_queue": {
+      "pt-br": "Criar nova fila",
+      "en": "Create new queue",
+      "es": "Crear nueva cola"
+    },
+    "queue_name": {
+      "pt-br": "Nome da fila",
+      "en": "Queue name",
+      "es": "Nombre de cola"
+    },
+    "queue_name_placeholder": {
+      "pt-br": "Exemplo: Pagamentos",
+      "en": "Example: Payments",
+      "es": "Ejemplo: Pagos"
+    },
+    "queue_title": {
+      "pt-br": "Filas em",
+      "en": "Queues in",
+      "es": "Cola en"
+    },
+    "queue_edit": {
+      "pt-br": "Editar",
+      "en": "Edit",
+      "es": "Editar"
+    },
+    "queue_table_name": {
+      "pt-br": "Nome",
+      "en": "Name",
+      "es": "Nombre"
+    },
+    "quantity_agents": {
+      "pt-br": "Qtd de agentes",
+      "en": "Number of agents",
+      "es": "Cantidad de agentes"
+    },
+    "visualize": {
+      "pt-br": "Visualizar",
+      "en": "Visualize",
+      "es": "Para ver"
```

Comment:
This traduction is already available as `view` in this file, so there is no need to repeat it here.